templates/watchyourlan.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>WatchYourLAN</Name>
+  <Repository>aceberg/watchyourlan</Repository>
+  <Registry>https://hub.docker.com/r/aceberg/watchyourlan</Registry>
+  <Network>host</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/aceberg/WatchYourLAN</Support>
+  <Project>https://github.com/aceberg/WatchYourLAN</Project>
+  <Overview>Lightweight network IP scanner with web GUI.</Overview>
+  <Category>Tools: Network:Management</Category>
+  <WebUI>http://[IP]:[PORT:8840]</WebUI>
+  <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/watchyourlan.xml</TemplateURL/>
+  <Icon>https://raw.githubusercontent.com/aceberg/WatchYourLAN/main/assets/logo.png</Icon>
+  <Requires>Network Type needs to be set to Host.</Requires>
+  <Config Name="Config" Target="/data" Default="" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/wyl</Config>
+  <Config Name="Timezone" Target="TZ" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
+  <Config Name="Web UI" Target="GUIPORT" Default="8840" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false">8840</Config>
```

Comment:
We probably don't need to change the internal web UI port. Can we remove this and just work with the assumption that the container uses its default 8840?

Reply:
I've removed it. Would the user then change the port via Advanced View - WebUI?

Reply:
Sorry, I should have been clear. Use a port mapping rather than an environmental variable

Reply:
Hmm, I've tried `<Config Name="GUIPORT" Target="8840" Default="8840" Mode="tcp" Description="" Type="Port" Display="always" Required="false" Mask="false">8840</Config>` but changing the port number doesn't seem to do anything. Also because it's in Host Network Type the `Container Port:` option isn't available.

Reply:
Ah, sorry, I missed that. Okay, if it's Host Mode, then yes, the mapping won't do anything. Probably best to re-add the original environmental variable method and simply account for the potential change in the Web UI.

@Roxedus do you know how to have the Web UI port account for a Host Network port?

Reply:
It doesnt, we just have to ignore the lint error for that on host networks

Reply:
Ok I've re-added in the original environmental variable method as well as added a bit more info to the Additional Requirements. Feel free to edit it if you need.

Perhaps I should also add that if they change the port then they should edit the WebUI: ? The port can also be edited from the Config tab in WatchYourLAN. So maybe I add info about that to the Additional Requirements? If the port gets edited from the Config tab but it's different to the environmental variable then the environmental variable will orderride it when it next gets restarted. So another option would be to remove the environmental variable completely and point them to changing it via the Config tab. But if they have a port conflict when they first install it then it could be difficult.